scene_graph/vrd_tutorial.py

```diff
@@ -155,36 +179,33 @@ def LF_area(x):
 L_train = applier.apply(train_df)
 L_valid = applier.apply(valid_df)
 
-# %%
+# +
 from snorkel.labeling import LFAnalysis
 
 Y_valid = valid_df.label.values
 LFAnalysis(L_valid, lfs).lf_summary(Y_valid)
+# -
 
-# %% [markdown]
 # ## 3. Train Label Model
 # We now train a multi-class `LabelModel` to assign training labels to the unalabeled training set.
 
-# %%
+# +
 from snorkel.labeling import LabelModel
 
 label_model = LabelModel(cardinality=3, verbose=True)
 label_model.fit(L_train, seed=123, lr=0.01, log_freq=10, n_epochs=100)
+# -
 
-# %% [markdown]
 # We use [F1](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html) Micro average for the multiclass setting, which calculates metrics globally by counting the total true positives, false negatives and false positives.
 
-# %%
 label_model.score(L_valid, Y_valid, metrics=["f1_micro"])
 
-# %% [markdown]
 # ## 4. Train a Classifier
 # You can then use these training labels to train any standard discriminative model, such as [an off-the-shelf ResNet](https://github.com/KaimingHe/deep-residual-networks), which should learn to generalize beyond the LF's we've developed!
 
-# %% [markdown]
 # #### Create DataLoaders for Classifier
 
-# %%
+# +
 from snorkel.classification import DictDataLoader
 from scene_graph.model import FlatConcat, SceneGraphDataset, WordEmb, init_fc
 
```

Comment:
can we handle the below `TRAIN_DIR` assignment in an `if sample` now?

Reply:
good catch!